api/base/permissions.py

```diff
@@ -53,6 +53,11 @@ def _get_scopes(self, request, view):
             except AttributeError:
                 raise ImproperlyConfigured('TokenHasScope requires the view to define the '
                                            'required_read_scopes attribute')
+            assert isinstance(view.required_read_scopes, list), 'The required_read_scopes must be a list of CoreScopes'
```

Comment:
Strictly speaking, this can also be an iterable. (I'm totally ok with the scopes field being a tuple instead of a list)

There is a relevant function `is_iterable_but_not_string`, though it inexplicably lives in `website.util.sanitize`...